ci_watson/plugin.py

```diff
@@ -30,7 +30,7 @@ def pytest_addoption(parser):
     # Choose to test under dev or stable
     parser.addoption(
         "--env",
-        choices=['dev', 'stable'],
+        choices=['dev', 'stable', 'None'],
```

Comment:
What does `'None'` do? Does it have to be capitalized? The capitalization is inconsistent with the other two allowed options.

Reply:
I would prefer that the internals set any internal variable that depends on `--env` to `None` or `''` if needed when the parser option isn't used.  I.e. for JWST, we don't have different environments.  So we would not use the parser option in running `pytest` either locally or in Jenkins, so it would be good to have the default to handle that.

Reply:
For practical reason, `jwst` should always use "dev", which is the default. Personally, I don't see any need for "stable" but maybe it would come in handy in the future. I am actually not sure how "None" is used.

Reply:
The option 'None' allows a repo to not use the envopt as the top most branch of the repo, whereas HSTCAL and DrizzlePac both have support for 'dev' in their repos now.  The decision was handed to me that JWST would not, at this point, use this option, and without this implementation of "None" for envopt, there was no way to get the interface in ci_watson to work with the JWST repo.

Reply:
I thought about this a little and think "none" is not needed. If you don't use `envopt`, you don't need to set any `self.env = ...` in your test class. The fixture being there does not mean you are obligated to use it.

Reply:
Preferably, the `jwst` tests would just not use the `pytest --env` cli option at all, since it is not needed.  And it should just work out of the box with that.

Reply:
> it should just work out of the box with that

I think it already does, without this change. Even if default is set to "dev", if you don't use it, it does not matter to you.

Reply:
We can just add a `dev` level to the jwst artifactory repo.  One extra directory level is preferable, as it gives us flexibility in the future to test against a stable branch if needed.